sdk_lib/make_chroot.sh

```diff
@@ -154,6 +159,13 @@ init_setup () {
    ln -sf "${CHROOT_TRUNK_DIR}/src/third_party/portage-stable" \
      "${FLAGS_chroot}"/"${PORTAGE_STABLE_OVERLAY}"
 
+   if [[ -n "${FLAGS_extra_portdir_overlay_path}" ]]; then
+     if [[ -n "${FLAGS_extra_portdir_overlay_name}" ]]; then
+       ln -sf "${CHROOT_TRUNK_DIR}"/"${FLAGS_extra_portdir_overlay_path}" \
```

Comment:
How about do this in update_chroot so you can an overlay after the initial chroot creation.